packages/odyssey-react-mui/src/properties/translations/odyssey-react-mui_fr.properties

```diff
@@ -0,0 +1,7 @@
+field.error.screen.reader.text = Erreur
+field.label.optional.text = Optionnel
+severity.success = succès
+severity.warning = avertissement
+severity.info = info
+severity.error = erreur
+toast.close.text = fermé
```

Comment:
This is just temporary for your own testing right? Once we merge this in, there'll be some config file from Globalization that tells them how to add properties files?

Reply:
![Screen Shot 2023-05-10 at 2 35 03 PM](https://github.com/okta/odyssey/assets/72201855/e3fc704a-4f6e-4671-a35b-05448cab474b)

@KevinGhadyani-Okta

Reply:
For testing purposes, if I DO have time, I will add the ok_PL (pseudoloc) script/language to this.